### PR TITLE
fix: broken links 🐛

### PR DESCRIPTION
**Pull request by kdheepak**
The issue here is when astro sees `src/content/docs/path/to/filename.md` it renders `path/to/filename/index.html`.

In `path/to/filename.md`, linking to `path/to/filename2.md` is usually done by `./filename2.md` but for the path to work in the web it has to be `./../filename`.

An alternative solution here would be to have _every_ file named `index.md` in the appropriate folders. Then the relative paths will always match correctly.

### CODE REVIEW

**Comment by kdheepak** [comment]
Ideally astro would just figure this out when it sees links in a markdown file. Alternatively, we can write a remark plugin to do this ( I think I've already written one for this in the past when my personal blog was in svelte ).

**Comment by kdheepak** [comment]
It's weird that cloudflare didn't post a preview here? I see the successful build on their website though.

File: astro.config.mjs
Comment (joshka) [inline astro.config.mjs:1]: all links need to end in / for this to work correctly

**Comment by joshka** [comment]
> It's weird that cloudflare didn't post a preview here? I see the successful build on their website though.

guessing it's the one build at a time thing?

**Pull request merged**